data-pipeline/tests/unit/pre_processing/test_maintained_schools.py

```diff
@@ -103,12 +101,13 @@ def test_map_school_attrs():
     assert {"Email", "HeadEmail", "Trust Name"}.issubset(list(actual.columns))
 
 
+@pytest.mark.skip("TODO: needs updating to reflect phase-type mappings.")
```

Comment:
I've skipped any phase-type-related tests for now: these should, however, be updated ASAP when we're confident in the changes.

Reply:
Restored.